WordPress/WordPressHomeWidgetToday/Model/TodayWidgetContent.swift

```diff
@@ -0,0 +1,12 @@
+import WidgetKit
+
+
+struct TodayWidgetContent: TimelineEntry {
+    var date = Date()
+    // TODO - TODAYWIDGET: see if this needs to be LocalizedStringKey
```

Comment:
If this is the site title as chosen by the user, there's no need to localize (and I think we can't even if we wanted to).

Reply:
👍